pkg/arvo/sys/hoon.hoon

```diff
@@ -821,14 +832,14 @@
   ::
 ++  bex                                                 ::  binary exponent
   ~/  %bex
-  |=  a=@
+  |=  a=bloq
```

Comment:
Disagree, if I e.g. `(bex 10)` I'm probably not using it as a stride size, and if I `(dec (bex (bex 5)))` the two bexes definitely have different types - hence unopinionated `@`.

Reply:
`bloq` is just an alias for `@`, communicating that the atom is used as a power of two -- that example is the same as the `(met 0 (met 0 atom))` in `+mat`.

Reply:
`met` doesn't produce `bloq`! And indeed, the outer one is a bloq but the inner one isn't, `a blocksize is the power of 2 size of an atom. ie, 3 is a byte as 2^3 is 8 bits.` - bloq implies 2^2^n not just any n used as a power of 2.